src/FINDOLOGIC/Export/Data/Image.php

```diff
@@ -50,7 +52,7 @@ public function getUrl(): string
 
     private function setUrl(string $url): void
     {
-        $url = DataHelper::checkForEmptyValue($url);
+        $url = DataHelper::checkForEmptyValue('image', $url);
```

Comment:
Can we implement the `NameAwareValue` class here as well and access the name via the `getValueName()` method instead?